lib/data_update_scripts/20210420050256_move_mascot_settings.rb

```diff
@@ -0,0 +1,18 @@
+module DataUpdateScripts
+  class MoveMascotSettings
+    SETTINGS = %w[
+      footer_image_height
+      footer_image_url
+      footer_image_width
+      image_description
+      image_url
+    ].freeze
+
+    def run
```

Comment:
@mstruve [raised a point](https://github.com/forem/forem/pull/13403#discussion_r619183273) in the Settings::Community PR about this - the update script runs _after_ deploy, while migrations run _before_ deployed code is reloaded - is there a sequencing concern at all here (if we can't find any items in the settings_mascots table and it's just nil for 10 minutes, that's not a problem)?

I assume it's the same what-if scenario here as it was there.

Reply:
If you don't mind I'll only answer this question once: https://github.com/forem/forem/pull/13403#discussion_r619924629

Reply:
Ok, I'm going to make use of my funky time zone and just sneakily deploy thus in the middle of the night for the US and Europe so the 10 minutes of weird data are a bit less of an issue.